api/src/main/resources/workbench-api.yaml

```diff
@@ -0,0 +1,6369 @@
+            default: false
+  "/v1/{workspaceNamespace}/{workspaceId}/data-set/exportToNotebook":
+    post:
+      tags:
+      - dataSet
+      description: 'Exports a data set query to a notebook
+
+'
+      operationId: exportToNotebook
+      parameters:
+      - "$ref": "#/parameters/workspaceNamespace"
+      - "$ref": "#/parameters/workspaceId"
+      - in: body
+        name: dataSetExportRequest
+        required: true
+        schema:
+          "$ref": "#/definitions/DataSetExportRequest"
+      responses:
+        200:
+          description: Data set has been exported correctly
+          schema:
+            "$ref": "#/definitions/EmptyResponse"
+        400:
+          description: Invalid data set request
+          schema:
+            "$ref": "#/definitions/ErrorResponse"
+  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/data-set/preview":
+    post:
+      tags:
+      - dataSet
+      description: 'Preview data set
+
+'
+      operationId: previewDataSetByDomain
+      parameters:
+      - "$ref": "#/parameters/workspaceNamespace"
+      - "$ref": "#/parameters/workspaceId"
+      - in: body
+        name: dataSetPreviewRequest
+        required: true
+        schema:
+          "$ref": "#/definitions/DataSetPreviewRequest"
+      responses:
+        200:
+          description: 'A SQL query for each domain in the Data Set
+
+'
+          schema:
+            "$ref": "#/definitions/DataSetPreviewResponse"
+  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/data-set/domainValues/{domain}":
+    get:
+      tags:
+      - dataSet
+      description: 'Returns all column names/values for a given domain.
+'
+      operationId: getValuesFromDomain
+      parameters:
+      - "$ref": "#/parameters/workspaceNamespace"
+      - "$ref": "#/parameters/workspaceId"
+      - in: path
+        name: domain
+        required: true
+        type: string
+      responses:
+        200:
+          description: the values/column names in the domain.
+          schema:
+            "$ref": "#/definitions/DomainValuesResponse"
+  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/getDataTableQuery":
+    parameters:
+    - "$ref": "#/parameters/workspaceNamespace"
+    - "$ref": "#/parameters/workspaceId"
+    post:
+      tags:
+      - cohorts
+      consumes:
+      - application/json
+      description: 'Translates a data table specification into a SQL query to run
+        against the CDR.
+
+'
+      operationId: getDataTableQuery
+      parameters:
+      - in: body
+        name: request
+        description: a query specification for a data table
+        schema:
+          "$ref": "#/definitions/DataTableSpecification"
+      responses:
+        200:
+          description: the query to run against the CDR to retrieve the data
+          schema:
+            "$ref": "#/definitions/CdrQuery"
+  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/getCohortAnnotations":
+    parameters:
+    - "$ref": "#/parameters
```

Comment:
This comment was dropped, it seems meaningful:

`      # These property names should match what is in ModifyParticipantCohortAnnotationRequest`